lib/src/transformations/enum_extensions.dart

```diff
@@ -0,0 +1,5 @@
+/// Provides extensions for [Enum] type
+extension EnumExtensions on Enum {
+  /// Returns an human readable value of this enum value.
```

Comment:
Nit: please correct the grammar here to "a human-readable"

Reply:
Hey, i have updated with those changes